rasa/nlu/model.py

```diff
@@ -211,6 +226,138 @@ def train(self, data: TrainingData, **kwargs: Any) -> "Interpreter":
 
         return Interpreter(self.pipeline, context)
 
+    def train_in_chunks(
+        self,
+        training_data: TrainingDataFull,
+        train_path: Path,
+        number_of_chunks: int,
+        fixed_model_name: Optional[Text] = None,
+    ) -> Tuple["Interpreter", Text]:
+        """Trains the underlying pipeline in chunks using the provided training data.
+
+        Args:
+            training_data: The training data.
+            train_path: The training path to use.
+            fixed_model_name: The model name to use.
+            number_of_chunks: The number of chunks to use.
+
+        Returns:
+            The trained interpreter and the path to the persisted model.
+        """
+        import tempfile
+        from rasa.nlu.utils.hugging_face.hf_transformers import HFTransformersNLP
+        from rasa.nlu.utils.mitie_utils import MitieNLP
```

Comment:
Does this mean any pipeline would need Mitie dependencies?

Reply:
what do you mean?

Reply:
I was worried that this would require the mitie dependency even if you don't use mitie in your pipeline. But seems it doesn't. False Alarm :)